### PR TITLE
Use openpyxl

### PR DESCRIPTION
**Pull request by aschroed**

### CODE REVIEW

**Comment by aschroed** [comment]
I modified several things to get openpyxl to work for both import_data and get_field_info.
I also updated several tests - some of which were quite wonky before - many tests are actually not unit tests in that they test multiple functions in a single test but that's how Koray wrote em.  
It's unclear to me if submitting Boolean fields ever worked correctly as we really didn't have any that would be submitted (and still have very few) but I believe it works as it should now.  I tested posting InvidualHuman items on webdev with protected either True or False or empty and got the expected results.  Some aspects of this are tested in unit tests.

File: tests/test_import_data.py [hunk 1]
```diff
@@ -298,12 +308,20 @@ def test_fix_attribution(connection_mock):
...
+def test_digest_xlsx(workbooks):
+    WORKBOOK_DIR = './tests/data_files/workbooks/'
+    for fn, workbook in workbooks.items():
+        book, sheets = imp.digest_xlsx(WORKBOOK_DIR + fn)
+        assert sheets == workbook.sheetnames
+        for sheet in sheets:
+            assert book[sheet].max_row == workbook[sheet].max_row
+            assert book[sheet].max_column == workbook[sheet].max_column
+
+
```
Comment (andreacosolo) [inline tests/test_import_data.py:321]: Here and in the following tests the @pytest.mark was removed. It doesn't seem necessary, I just want to make sure it's intentional.

**Review by andreacosolo — COMMENTED** [review]
Looks good, I left a couple minor comments. I have tested only get_field_info, will try to use import_data from this branch in the upcoming submissions.

In addition to specific comments, I noticed that there are many xls files in the repo, including those in the Data_Files directory. Are they still used by any script? I'm not sure whether converting them to xlsx or removing completely.

File: wranglertools/get_field_info.py [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
 import attr
-import xlwt
+import openpyxl
 import sys
 import json
 
 
 EPILOG = '''
-    To create an xls file with sheets to be filled use the example and modify to your needs.
+    To create an test_digest_xlsx file with sheets to be filled use the example and modify to your needs.
```
[the remark this answers is not in the document]
Comment (aschroed) [reply wranglertools/get_field_info.py:13]: mistake - damn you auto complete - fixed

File: tests/test_get_field_info.py [hunk 3]
```diff
@@ -345,84 +347,101 @@ def test_get_uploadable_fields_mock(connection_mock, mocker, returned_vendor_sch
 def xls_to_list(xls_file, sheet):
     """To compare xls files to reference ones, return a sorted list of content."""
     return_list = []
```
[the remark this answers is not in the document]
Comment (aschroed) [reply tests/test_get_field_info.py:349]: fixed

File: tests/test_import_data.py [hunk 4]
```diff
@@ -98,17 +102,23 @@ def test_reader_with_sheetname(vendor_raw_xls_fields):
 
 
 @pytest.mark.file_operation
-def test_reader_wrong_sheetname():
-    readxls = imp.reader('./tests/data_files/Vendor.xls', 'Enzyme')
-    list_readxls = list(readxls)
-    assert list_readxls == []
+def test_reader_wrong_sheetname(capsys, workbooks):
```
[the remark this answers is not in the document]
Comment (aschroed) [reply tests/test_import_data.py:105]: removed

File: tests/test_import_data.py (hunk 1 above)
Comment (aschroed) [reply tests/test_import_data.py:321]: it was as a fixture is used and no longer requires opening a file directly in the test.

File: wranglertools/get_field_info.py [hunk 5]
```diff
@@ -96,7 +96,7 @@ def getArgs():  # pragma: no cover
                         help="Do not include enums (or suggestions) for fields.")
     parser.add_argument('--outfile',
                         default='fields.xls',
```
Comment (andreacosolo) [inline wranglertools/get_field_info.py:98]:
```suggestion
                        default='fields.xlsx',
```

**Review by andreacosolo — APPROVED** [review]
Tested also `import_data`, works as expected.

**Pull request merged**